ci/provision-cluster.sh

```diff
@@ -14,7 +14,7 @@ fi
   cp examples/cluster.ini ./
   sed -i '/nomad_meta/a vmck_worker = true' cluster.ini
   sed -i '/^run_jobs =.*/c\run_jobs = dnsmasq' cluster.ini
-  bin/docker.sh --rm --pull
+  bash bin/docker.sh --rm --pull
```

Comment:
Q: Should the ```bin/docker.sh``` file be executable? (I do not know if at the moment is)

Reply:
`bin/docker.sh` is executable. If I run it with bash, for files that end with `\r\n` as new line, it ignores the `\r` and the script runs as expected.

Reply:
And shouldn't we remove the executable right from the ```bin/docker.sh``` if we call the file like this?

Reply:
Nah, leave it, it's meant to be executable, we'll probably want to call it directly sometime in the future.